bodhi/client/__init__.py

```diff
@@ -427,6 +427,7 @@ def comment(update, text, karma, user, password, url, **kwargs):
 @updates.command()
 @click.option('--staging', help='Use the staging bodhi instance',
               is_flag=True, default=False)
+@click.option('--arch', help='Specify arch of packages to download instead current arch')
```

Comment:
Let's tack on this at the end:

. "all" will retrieve packages from all architectures